manual/spotify/spotify.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>spotify</id>
+    <version>1.0.49.125</version>
+    <title>Spotify</title>
+    <authors>Spotify</authors>
+    <projectUrl>http://spotify.com</projectUrl>
+    <owners>Boyan Kostadinov (boyank@gmail.com)</owners>
```

Comment:
don't include emails in the owners field (I believe that is a requirement on chocolatey).
also, you're missing the users `chocolatey` (should be listed as the first owner), as well as `ferventcoder`